### DISCUSSION TITLE
Access Denied from Swiper Studio

### DISCUSSION THREAD

**Discussion by alibey-10** [nolimits4web/swiper#5997] [Q&A]
I tried to sign in, but I got "Access Denied" and "you do not have permission to sign in" 
Is it from Patreon or Swiper Studio?

page URL that I got error from `https://studio.swiperjs.com/api/auth/error?error=AccessDenied`

**Comment by nolimits4web** [comment]
What is your patreon email address you used for subscription?

**Reply by alibey-10** [reply]
ali.hussein.pre@gmail.com

**Comment by nolimits4web** [comment]
There is no Swiper patron with active subscription with such email address

**Reply by alibey-10** [reply]
I am trying to do it for the first time. there isn't a signup button.
